recipes/vcfsim/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "VCFSimulator-SamukLab" %}
+{% set version = "1.0.5.alpba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Pie115/VCFSimulator-SamukLab/archive/{{ version }}.tar.gz
+  sha256: 3930958c2e52f62e5dd9e685307ff03e5387ae80158339806152da7f7aaa57ac
+
+build:
+  entry_points:
+    - vcfsim=vcfsim.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
`noarch` cannot also involve compilation. The package repository appears to be consistent with `noarch`, so the compilation components should likely be omitted.

Reply:
Thank you for the help and suggestions, I will look into this ASAP.